http-core/src/main/scala/org/apache/pekko/http/scaladsl/model/HttpHeader.scala

```diff
@@ -99,10 +99,11 @@ object HttpHeader {
             case HeaderParser.RuleNotFound => ParsingResult.Ok(RawHeader(name, preProcessedValue), Nil)
           }
         case Failure(error) =>
-          val info = (error match {
-            case e: ParseError => parser.parseError(e)
-            case e             => parser.failure(e)
-          }).info
+          val info = error match {
+            case e: ParseError           => parser.parseError(e).info
+            case _: NullPointerException => ErrorInfo()
+            case e                       => parser.failure(e).info
```

Comment:
how about `case e if e.getMessage() == null => ???` ?

This is useful too.

Reply:
yeah, done